py3status/modules/net_rate.py

```diff
@@ -5,41 +5,52 @@
 Configuration parameters:
     all_interfaces: ignore self.interfaces, but not self.interfaces_blacklist
         (default True)
-    cache_timeout: how often we refresh this module in seconds (default 2)
-    devfile: location of dev file under /proc (default '/proc/net/dev')
-    format: format of the module output (default '{interface}: {total}')
-    format_no_connection: when there is no data transmitted from the
-        start of the plugin (default '')
-    hide_if_zero: hide indicator if rate == 0 (default False)
-    interfaces: comma separated list of interfaces to track (default '')
+    cache_timeout: how often we refresh this module in seconds
+        (default 2)
+    devfile: location of dev file under /proc
+        (default '/proc/net/dev')
+    format: format of the module output
+        (default '{interface}: {total}')
+    format_no_connection: when there is no data transmitted from the start of the plugin
+        (default '')
+    format_value: format to use for values
+        (default "[\?min_length=11 {value:.1f} {unit}]")
+    hide_if_zero: hide indicator if rate == 0
+        (default False)
+    interfaces: comma separated list of interfaces to track
+        (default [])
     interfaces_blacklist: comma separated list of interfaces to ignore
         (default 'lo')
-    precision: amount of numbers after dot (default 1)
+    si_units: use SI units
+        (default False)
+    thresholds: thresholds to use for colors
+        (default [(0, 'bad'), (1024, 'degraded'), (1024*1024, 'good')])
+    unit: unit to use. If the unit contains a multiplier prefix, only this
+        exact unit will ever be used
+        (default "B/s")
 
 Format placeholders:
     {down} download rate
     {interface} name of interface
     {total} total rate
     {up} upload rate
 
+Value placeholders:
```

Comment:
Change this and I think we are merge ready.  It'll be consistent with the way you have in sysdata